### ISSUE TITLE
No Progress output since webpack 2.x

### ISSUE THREAD

**Issue by IAMtheIAM** [webpack/webpack-dev-server#609]
Reporting: Bug

Issue: No progress bar or percentage output in red is being shown on console. This is all I get:
```
 http://localhost:4000/
webpack result is served from http://localhost:4000/
content is served from C:\Source\xxxx
Version: webpack 2.1.0-beta.22
Time: 33949ms
webpack: bundle is now VALID.
[default] Checking started in a separate process...
[default] Ok, 3.815 sec.
```
Command I'm running:
`cross-env NODE_ENV=development webpack-dev-server --config config/webpack.development.js --profile --progress --inline --hot --compress`

Running a `webpack` production build of:
`cross-env NODE_ENV=production webpack --config config/webpack.production.js --profile --progress --colors --display-cached --bail` does produce the progress output as expected. 

Note: the `cross-env` does not affect the webpack output to console, it is for other purposes.
What happened to the output for webpack dev server?

**Comment by SpaceK33z** [comment]
Thanks for your bug report. I traced it back to [this commit](https://github.com/webpack/webpack/commit/46b12b2095577391fbd88b6d29b7f0b858db4c5c), released in `webpack@2.1.0-beta.22`. I'm going to work on a fix now.

**Comment by IAMtheIAM** [comment]
You're welcome, I'm glad to help, thanks for working on it.
